semcore/select/src/Select.jsx

```diff
@@ -342,8 +342,8 @@ function Checkbox(providedProps) {
   const { className, style } = styles.cn('SOptionCheckbox', {
     size,
     'use:theme': resolveColor(theme),
-    checked: selected,
```

Comment:
I'm not sure about this. As I see in `styles` - there are different color for `checked` and `selected` checkboxes.

Reply:
As I see in [design guide](https://developer.semrush.com/intergalactic/components/select/select#multiselect) should be no difference between checked or selected states